REPORT.md

```diff
@@ -189,40 +507,41 @@ vainfo: Supported profile and entrypoints
 |TSCL_A_VIDYO_5|✔️|✔️|✔️|✔️|
 |TSCL_B_VIDYO_4|✔️|✔️|✔️|✔️|
 |TSKIP_A_MS_3|✔️|✔️|✔️|✔️|
-|TSUNEQBD_A_MAIN10_Technicolor_2|✔️|❌|❌|❌|
+|TSUNEQBD_A_MAIN10_Technicolor_2|☠|☠|❌|✔️|
 |TUSIZE_A_Samsung_1|✔️|✔️|✔️|✔️|
 |VPSID_A_VIDYO_2|✔️|✔️|✔️|✔️|
-|VPSSPSPPS_A_MainConcept_1|✔️|❌|❌|❌|
-|WP_A_MAIN10_Toshiba_3|✔️|❌|✔️|✔️|
+|VPSSPSPPS_A_MainConcept_1|❌|❌|❌|✔️|
+|WP_A_MAIN10_Toshiba_3|✔️|✔️|✔️|✔️|
 |WP_A_Toshiba_3|✔️|✔️|✔️|✔️|
 |WP_B_Toshiba_3|✔️|✔️|✔️|✔️|
-|WP_MAIN10_B_Toshiba_3|✔️|❌|✔️|✔️|
-|WPP_A_ericsson_MAIN10_2|✔️|❌|✔️|✔️|
+|WP_MAIN10_B_Toshiba_3|✔️|✔️|✔️|✔️|
+|WPP_A_ericsson_MAIN10_2|✔️|✔️|✔️|✔️|
 |WPP_A_ericsson_MAIN_2|✔️|✔️|✔️|✔️|
-|WPP_B_ericsson_MAIN10_2|✔️|❌|✔️|✔️|
+|WPP_B_ericsson_MAIN10_2|✔️|✔️|✔️|✔️|
 |WPP_B_ericsson_MAIN_2|✔️|✔️|✔️|✔️|
-|WPP_C_ericsson_MAIN10_2|✔️|❌|✔️|✔️|
+|WPP_C_ericsson_MAIN10_2|✔️|✔️|✔️|✔️|
 |WPP_C_ericsson_MAIN_2|✔️|✔️|✔️|✔️|
-|WPP_D_ericsson_MAIN10_2|✔️|❌|✔️|✔️|
+|WPP_D_ericsson_MAIN10_2|✔️|✔️|✔️|✔️|
 |WPP_D_ericsson_MAIN_2|✔️|✔️|✔️|✔️|
-|WPP_E_ericsson_MAIN10_2|✔️|❌|✔️|✔️|
+|WPP_E_ericsson_MAIN10_2|✔️|✔️|✔️|✔️|
 |WPP_E_ericsson_MAIN_2|✔️|✔️|✔️|✔️|
-|WPP_F_ericsson_MAIN10_2|✔️|❌|✔️|✔️|
+|WPP_F_ericsson_MAIN10_2|✔️|✔️|✔️|✔️|
 |WPP_F_ericsson_MAIN_2|✔️|✔️|✔️|✔️|
-|TOTAL|147/147|124/147|136/147|134/147|
+|-|-|-|-|-|
+|TOTAL|143/147|145/147|143/147|147/147|
 
 
-## H.264/AVC
+### H.264/AVC Intel
```

Comment:
Can you add a subtitle that specifies the test suite. In the future we might want to add results for other test suites